app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="container d-flex flex-column justify-content-between vh-100 bg-light px-0">  
+  <%= form_for [@category, @expense] do |f| %>
+    <div class="d-flex justify-content-between align-items-center header">
+      <%= link_to category_expenses_path(@category) do %>
+        <span class="bi bi-arrow-left" style="font-size:24px;"></span>
+      <% end %>
+      <span class="text-uppercase">New Transaction</span>
+      <%= f.submit "Save", class: "btn text-white border-0" %>  
+    </div>
+
+    <fieldset class="mb-3">
+      <%= f.text_field :name, placeholder: "Name", required: true, class: "form-control rounded-0 shadow-none" %>
+      <%= f.number_field :amount, placeholder: "Amoumnt", required: true, class: "form-control rounded-0 shadow-none" %>
+
+      <%= f.label :category %>
+      <%= f.select :category, options_for_select(current_user.categories.map { |c| [c.name, c.id] }, selected: @category.id), { prompt: "Select a category" }, { required: true } %>
+      <%= f.hidden_field :author_id, value: current_user.id %>
+      <%= f.hidden_field :category_id, value: @category.id %>
```

Comment:
- [ ] For selecting category while creating new expenses. When I try to select a category other than the one I'm creating the expense from, it selects the very same category anyway. This is due to the ERD association issue described above. 🙏 
_Please take a look at the ordered pictures for more details._

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/235305899-39681d3f-523d-4540-be66-a4e7176039bd.png)
![Screenshot (2)](https://user-images.githubusercontent.com/47625407/235305894-fdaf1f0b-1e83-4fb7-8918-665cec30853c.png)
![Screenshot (3)](https://user-images.githubusercontent.com/47625407/235305897-a32fe2af-2f5d-48aa-84be-1fa29158f8b5.png)

Reply:
well resolved! ✔️